### PR TITLE
Adds batch commands for Signal, Cancel, and Terminate

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?

[Preview](https://temporal-documentation-git-edu-1114.preview.thundergun.io/cli/batch)

Adds CLI examples for the batch commands.
Includes a note about being careful when executing them.

### CODE REVIEW

File: docs-src/cli/batch/index.md [hunk 1]
```diff
@@ -16,9 +16,46 @@ The List Filter identifies the Workflow Executions in the Batch Job; the Batch t
...
 There are three types of Batch Jobs:
 
-- Signal: sends a [Signal](/concepts/what-is-a-signal) to the Workflow Executions specified by the List Filter.
 - Cancel: cancels the Workflow Executions specified by the List Filter.
+- Signal: sends a [Signal](/concepts/what-is-a-signal) to the Workflow Executions specified by the List Filter.
 - Terminate: terminates the Workflow Executions specified by the List Filter.
 
+Batch operations can affect multiple Workflows simultaneously.
+Depending on your needs, you might want to send Signals to running Workflows, Cancel them, or even Terminate them entirely.
+Below, are examples on these commands using Temporal's CLI for each type of Batch operation.
```
Comment (flossypurse) [inline docs-src/cli/batch/index.md:25]:
```suggestion
Below are examples of how to use the Temporal CLI for each type of Batch operation.
```

File: docs-src/cli/batch/index.md [hunk 2]
```diff
@@ -16,9 +16,46 @@ The List Filter identifies the Workflow Executions in the Batch Job; the Batch t
...
+
+Using these commands will directly impact the Workflows you target, so it's important to use them judiciously.
```
Comment (flossypurse) [inline docs-src/cli/batch/index.md:27]:
```suggestion
These commands will directly impact the Workflows you target, so it's important to use them judiciously.
```

File: docs-src/cli/batch/index.md [hunk 3]
```diff
@@ -16,9 +16,46 @@ The List Filter identifies the Workflow Executions in the Batch Job; the Batch t
...
+
+**What Are the CLI Commands for Batch operations in Temporal Workflows?**
```
Comment (flossypurse) [inline docs-src/cli/batch/index.md:29]: Maybe move this to the top and rephrase to: "What batch operations can be performed by the Temporal CLI?"

File: docs-src/cli/batch/index.md [hunk 4]
```diff
@@ -16,9 +16,46 @@ The List Filter identifies the Workflow Executions in the Batch Job; the Batch t
...
+
+You can use the `--query` flag to filter the Workflow Executions to be affected by the Batch Job.
```
Comment (flossypurse) [inline docs-src/cli/batch/index.md:31]: I might add that the value of `--query` is actually a List Filter -> and link to that.

File: docs-src/cli/batch/index.md [hunk 5]
```diff
@@ -16,9 +16,46 @@ The List Filter identifies the Workflow Executions in the Batch Job; the Batch t
...
+
+To Cancel a Workflow
```
Comment (flossypurse) [inline docs-src/cli/batch/index.md:33]:
```suggestion
To Cancel a Workflow:
```

**Review by flossypurse — APPROVED** [review]

File: docs-src/cli/batch/index.md [hunk 6]
```diff
@@ -16,9 +16,46 @@ The List Filter identifies the Workflow Executions in the Batch Job; the Batch t
...
+
+```command
+temporal workflow cancel \
+  --query 'ExecutionStatus = "Running" AND WorkflowType="YourWorkflow"' \
+  --reason "Testing"
+```
+
+To Signal a Workflow:
```
Comment (taonic) [inline docs-src/cli/batch/index.md:41]: Given the batch operation, would we consider to use plural here?

**Pull request merged**